full-service/src/db/transaction_log.rs

```diff
@@ -1887,7 +1883,4 @@ mod tests {
 
         assert_eq!(tx_log.tx, expected_tx_log.tx);
     }
-
-    // FIXME: test_log_submitted for recovered
-    // FIXME: test_log_submitted offline flow
```

Comment:
Do these tests exist?

Reply:
No, they don't, but I don't think they are applicable. This is from a very long time ago. I need to go through and clean up all of the TODO's and FIXME's, there are some that just don't apply anymore to the codebase.